.github/workflows/rebuild-released-images.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
           fi
   
   build-and-publish-image:
+    environment: release
```

Comment:
Do we support signing in daily rebuilds?

Reply:
Yes, it is the other yaml, `rebuild-released-images.yaml. But that one I cannot test until I merge it. Code is mostly the same as for a regular release.

Reply:
Perfect! thanks!